File: DSA/Linear.java
public class Linear {
       public static void main(String args[]) {
              int arr[] = { 7, 8, 9, 5, 3, 2, 10, 13 };
              int item = 81;
              Search list = new Search();
              list.perform_search(arr, item);
       }

}

class Search {
       public void perform_search(int arr[], int item) {
              int flag = 0;
              for (int i = 0; i < arr.length; i++) {
                     if(arr[i] == item) {
                            flag = 1;
                     break;
                     }
              }
              if(flag == 1){
                     System.out.println("Item found.");
              }
              else{
                     System.out.println("Item not found.");
              }
       }
}

File: DSA/linear.py
#linear search | sequential search | simple search
arr = []
flag = 0
n = int(input("enter the no elements you want:"))
for i in range(n):
    x = int(input("Enter the element:"))
    arr.append(x)
print("The array is:")
print(arr)
item  = int(input("Enter the item you are looking for:"))
for i in arr:
    if i == item:
        print("Item found at ",arr.index(i)," index")
        flag = 1
        break
if(flag == 0):
    print("Item is not in the array")   

# arr = []
# flag = 0
# n = int(input("enter the no elements you want:"))
# for i in range(n):
#     x = input("Enter the element:")
#     arr.append(x)
# print("The array is:")
# print(arr)
# item  = input("Enter the item you are looking for:")
# for i in arr:
#     if i == item:
#         print("Item found at ",arr.index(i)," index")
#         flag = 1
#         break
# if(flag == 0):
#     print("Item is not in the array") 

    